.travis.yml

```diff
@@ -118,8 +118,24 @@ jobs:
 
                 # Composer: boost installation
                 - composer global show hirak/prestissimo -q || travis_retry composer global require $DEFAULT_COMPOSER_FLAGS hirak/prestissimo
+
+                # Require PHPUnit 8
+                - composer require --dev --no-update phpunit/phpunit:^8
+
+                # Install PCOV
+                - |
+                    git clone --branch=release --depth=1 https://github.com/krakjoe/pcov
```

Comment:
`--single-branch` ?
maybe pinpoint to tag instead of branch, that will change one day ?

Reply:
`git clone --single-branch --branch=v1.0.1 --depth=1 https://github.com/krakjoe/pcov` ?
ref https://github.com/sebastianbergmann/php-code-coverage/pull/673

Reply:
[Docs](https://github.com/krakjoe/pcov/blob/develop/INSTALL.md#installation) says:

> use `release` branch for last stable released version

Reply:
I cannot see any improvement with `--single-branch` - `--depth=1` already does the job.

Reply:
so, release would be last stable 1.x or last stable 2.x ? you don't know. let us rely on BC versioning, and not random stable